packages/terra-action-header/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ ChangeLog
 
 Unreleased
 ----------
+### Changed
+* Changed buttons' variant to `Utility`.
```

Comment:
we should add a line mentioned we dropped button-group dependency

Reply:
done.